src/files.rs

```diff
@@ -274,7 +274,9 @@ impl Files {
               ":"
             }
             div class="payment-request" {
-              (invoice.payment_request)
+              a href={"lightning:" (invoice.payment_request)} {
```

Comment:
Maud uses `(…)` to delimit rust expressions to insert into the HTML. So I think this should be:

```suggestion
              a href=(format!("lightning: {}", invoice.payment_request)) {
```

I didn't test it though.